### PR TITLE
Fix not zipping config files, remove build-extras

### LINKED ISSUE #37: Building on phonegap runs into error
When using the new package (in combination with the new Push Notification setup) I run in to an issue when building the app on Phonegap:

image

The file is there as far as I can tell, below is a screenshot of my .zip contents:

image

Complete PGB log file:
```
    Build Date: 2019-01-08 12:38:29 +0000
--------------------------------------------------------------------------------
PREPARE OUTPUT
--------------------------------------------------------------------------------
Android Studio project detected
Generating platform-specific config.xml from defaults for android at /app/src/main/res/xml/config.xml
Merging project's config.xml into platform-specific android config.xml
Merging and updating files from [www, ../platform_www] to ../app/src/main/assets/www
  mkdir ../app/src/main/assets/www/cordova-js-src
  mkdir ../app/src/main/assets/www/cordova-js-src/android
  copy  ../platform_www/cordova-js-src/android/nativeapiprovider.js ../app/src/main/assets/www/cordova-js-src/android/nativeapiprovider.js (new file)
  copy  ../platform_www/cordova-js-src/android/promptbasednativeapi.js ../app/src/main/assets/www/cordova-js-src/android/promptbasednativeapi.js (new file)
  copy  ../platform_www/cordova-js-src/exec.js ../app/src/main/assets/www/cordova-js-src/exec.js (new file)
  copy  ../platform_www/cordova-js-src/platform.js ../app/src/main/assets/www/cordova-js-src/platform.js (new file)
  mkdir ../app/src/main/assets/www/cordova-js-src/plugin
  mkdir ../app/src/main/assets/www/cordova-js-src/plugin/android
  copy  ../platform_www/cordova-js-src/plugin/android/app.js ../app/src/main/assets/www/cordova-js-src/plugin/android/app.js (new file)
  copy  ../platform_www/cordova.js ../app/src/main/assets/www/cordova.js (updated file)
  copy  www/css/index.css ../app/src/main/assets/www/css/index.css (updated file)
  delete ../app/src/main/assets/www/img/cordova.png (no source)
  copy  www/img/error.png ../app/src/main/assets/www/img/error.png (new file)
  delete ../app/src/main/assets/www/img/logo.png (no source)
  copy  www/index.html ../app/src/main/assets/www/index.html (updated file)
  copy  www/js/bundle.js ../app/src/main/assets/www/js/bundle.js (new file)
  delete ../app/src/main/assets/www/js/index.js (no source)
  copy  www/settings.json ../app/src/main/assets/www/settings.json (new file)
Wrote out android application name "TESTkompas" to /app/src/main/res/values/strings.xml
android-versionCode not found in config.xml. Generating a code based on version in config.xml (5.0.0): 50000
Wrote out Android package name "nl.enexis.vhk" to /app/src/main/java/nl/enexis/vhk/TESTkompas.java
Updating icons at ../app/src/main/res
  copy  res/android/icon-72-hdpi.png ../app/src/main/res/mipmap-hdpi/icon.png (updated file)
  copy  res/android/icon-36-ldpi.png ../app/src/main/res/mipmap-ldpi/icon.png (updated file)
  copy  res/android/icon-48-mdpi.png ../app/src/main/res/mipmap-mdpi/icon.png (updated file)
  copy  res/android/icon-96-xhdpi.png ../app/src/main/res/mipmap-xhdpi/icon.png (updated file)
  copy  res/android/icon-144-xxhdpi.png ../app/src/main/res/mipmap-xxhdpi/icon.png (updated file)
  copy  res/android/icon-192-xxxhdpi.png ../app/src/main/res/mipmap-xxxhdpi/icon.png (updated file)
Updating splash screens at ../app/src/main/res
  copy  res/android/screen-hdpi-landscape.png ../app/src/main/res/drawable-land-hdpi/screen.png (updated file)
  copy  res/android/screen-ldpi-landscape.png ../app/src/main/res/drawable-land-ldpi/screen.png (updated file)
  copy  res/android/screen-mdpi-landscape.png ../app/src/main/res/drawable-land-mdpi/screen.png (updated file)
  copy  res/android/screen-xhdpi-landscape.png ../app/src/main/res/drawable-land-xhdpi/screen.png (updated file)
  copy  res/android/screen-xxhdpi-landscape.png ../app/src/main/res/drawable-land-xxhdpi/screen.png (updated file)
  copy  res/android/screen-xxxhdpi-landscape.png ../app/src/main/res/drawable-land-xxxhdpi/screen.png (updated file)
  copy  res/android/screen-xxxhdpi-portrait.png ../app/src/main/res/drawable-mdpi/screen.png (new file)
  copy  res/android/screen-hdpi-portrait.png ../app/src/main/res/drawable-port-hdpi/screen.png (updated file)
  copy  res/android/screen-ldpi-portrait.png ../app/src/main/res/drawable-port-ldpi/screen.png (updated file)
  copy  res/android/screen-mdpi-portrait.png ../app/src/main/res/drawable-port-mdpi/screen.png (updated file)
  copy  res/android/screen-xhdpi-portrait.png ../app/src/main/res/drawable-port-xhdpi/screen.png (updated file)
  copy  res/android/screen-xxhdpi-portrait.png ../app/src/main/res/drawable-port-xxhdpi/screen.png (updated file)
  copy  res/android/screen-xxxhdpi-portrait.png ../app/src/main/res/drawable-port-xxxhdpi/screen.png (updated file)
Updating resource files at ..
  copy  google-services.json ../app/google-services.json (new file)
Error: Source path does not exist: build-extras.gradle
    at updatePathInternal (/cordova/node_modules/cordova-common/src/FileUpdater.js:145:19)
    at /cordova/node_modules/cordova-common/src/FileUpdater.js:223:19
    at Array.forEach (native)
    at Object.updatePaths (/cordova/node_modules/cordova-common/src/FileUpdater.js:221:33)
    at updateFileResources (/cordova/lib/prepare.js:432:17)
    at /cordova/lib/prepare.js:51:9
    at _fulfilled (/cordova/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/cordova/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/cordova/node_modules/q/q.js:796:13)
    at /cordova/node_modules/q/q.js:604:44
```

### PR DESCRIPTION
**Pull request by mmehmetAliIzci**
Fixes an issue where configs were not copied to zip file.
Fixes an issue where missing build-extras.gradle would cause phonegap build to fail.

Fixes #37

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -110,6 +106,16 @@ module.exports = function(env) {
         })
     }
 
+    // Merge zip later because its not copying configs in if
```
Comment (smarki) [inline webpack.config.js]: if?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/mendix-hybrid-app-base",
-  "version": "4.1.4",
+  "version": "4.1.5",
```
Comment (smarki) [inline package.json]: No reason to up the version till we actually do a tag and release for it

File: webpack.config.js (hunk 1 above)
Comment (jogboms) [inline webpack.config.js]: Could there be more info on this?

File: webpack.config.js [hunk 3]
```diff
@@ -110,6 +106,16 @@ module.exports = function(env) {
...
+    config = webpack_merge(config,{
+        plugins: [
+            new ZipPlugin({
+                path: "../dist",
+                filename: utils.constructArchiveName(settings)
+              })
+        ] 
+    })
```
Comment (smarki) [inline webpack.config.js]: ;

**Review by smarki — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (jogboms) [reply package.json]: True.

File: webpack.config.js (hunk 3 above)
Comment (jogboms) [reply webpack.config.js]: Missed

**Review by smarki — APPROVED** [review]

**Pull request merged**